### DISCUSSION TITLE
Armor Of The True Assassin (BROTHERHOOD) [UR]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#24] [Card Balance]
Armor Of The True Assassin (BROTHERHOOD) [UR]
EQUIPMENT, UNIQUE, ARMOR, FACTION
The Warrior is a MORTIFICATOR and a SLAYER, and has FIRSTSTRIKE.

It's not a bad card but something is missing for an armor. 
:arrow_right: +2 to [A]

**Comment by TomekPL-1** [comment]
21-ric proposal +2A is good idea

I have idea about Mortificator key word. Like Michał said before:
"Mortificators are killers who excel in stealth and attacks from the shadow... most people never seen one and if they its often last thing they see.."

Mortificator key word- can ignore opponent FRONTLINE warriors

In table top doomtrooper mortificators could attack any warrior despite of affilation.

**Reply by 21-ric** [reply]
There is a lot of cards in the collection for Mortificator warriors.

![image](https://user-images.githubusercontent.com/83040170/118254253-2fd7a100-b4ab-11eb-9386-5f50f3b2c9a3.png)

![image](https://user-images.githubusercontent.com/83040170/118254340-48e05200-b4ab-11eb-8744-380972f1cb62.png)

![image](https://user-images.githubusercontent.com/83040170/118254431-61e90300-b4ab-11eb-8182-a67526b56f9e.png)

**Reply by framerate** [reply]
Yes but do we like that? There was basically an entire expansion for ONE subtype. 

Its almost as if Bryan didn't want people playing Inquisitors or Cardinals or anything else Brotherhood because Crenshaw was his favorite....

**Comment by Sol-au-fein** [comment]
+2A is appropriate. Considered it is Unique item.
